extendr-api/src/graphics/mod.rs

```diff
@@ -92,6 +177,52 @@ pub enum FontFace {
     SymbolFont,
 }
 
+impl LineEnd {
+    fn to_u32(&self) -> u32 {
+        match self {
+            Self::RoundCap => 1,
+            Self::ButtCap => 2,
+            Self::SquareCap => 3,
+        }
+    }
+}
+
+impl LineJoin {
+    fn to_u32(&self) -> u32 {
+        match self {
+            Self::RoundJoin => 1,
+            Self::MitreJoin => 2,
+            Self::BevelJoin => 3,
+        }
+    }
+}
```

Comment:
What about changing the names of constants to reduce repetition? I believe most of the time it will be accessed as `LineJoin::RoundJoin`, which might be more readable as `LineJoin::Round`.

Reply:
Agreed.